postProcess/Video3D.py

```
@@ -1,144 +1,382 @@
+        command (list): Command and arguments to execute
+
+    Returns:
+        str: Decoded stderr output containing geometric data
+
+    Raises:
+        subprocess.CalledProcessError: If the external command fails
+
+    Note:
+        Basilisk C utilities output data to stderr by convention
+    """
+    p = sp.Popen(command, stdout=sp.PIPE, stderr=sp.PIPE)
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)